common/remote/sockfile.cpp

```diff
@@ -3857,17 +4015,26 @@ class CRemoteDiskReadActivity : public CSimpleInterfaceOf<IRemoteActivity>, impl
                 throw MakeStringException(1, "Failed to open: '%s'", fileName);
         }
 
-        inputStream.setown(createFileSerialStream(iFileIO, startPos));
-        prefetchBuffer.setStream(inputStream);
-        prefetcher.setown(helper->queryDiskRecordSize()->createDiskPrefetcher());
+        if (canMatchAny)
```

Comment:
And this as well.  Should there be a assertex(canMatchany) at the start of the function?

Reply:
will remove